src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -15,12 +15,18 @@ public class AssemblyLoadContextTest
     {
         private const string TestAssembly = "System.Runtime.Loader.Test.Assembly";
 
+        [ActiveIssue(/* dotnet/coreclr */ 1187, PlatformID.Windows)] // dependency on coreclr behavior, waiting for new coreclr
```

Comment:
Nevermind, I already did put it in a different PR :). I'll just need to wait for https://github.com/dotnet/coreclr/pull/2015 to be submitted and wait for that to be published before submitting this one.